internal/capability/operand_cleanup.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // OperandCleanup removes the operand from the OCP cluster in the ca.namespace
-func (ca *capAudit) OperandCleanUp() error {
+func (ca *capAudit) OperandCleanUp(ctx context.Context) error {
```

Comment:
Small nit: no need to change the signature of these functions. `ca *capAudit` will already have the context. We can pass that context. Same for Operand_Install, Operator_Install, Operator_CleanUp

Reply:
Contexts should not be saved off like that. It should be passed in.

Reply:
@yashoza19 I think for context that sounds right. We will be passing context in multiple other places. So in this case if we change it, it will look weird. The standard is actually to pass the context. But I agree that changing function signatures because you've change a bit of data on an object is not practical. But that's not the case here.